src/model/AirLoopHVACOutdoorAirSystem.cpp

```diff
@@ -349,6 +355,9 @@ namespace model {
         } else if (boost::optional<WaterToAirComponent> comp = modelObject->optionalCast<WaterToAirComponent>()) {
           modelObjects.push_back(*comp);
           modelObject = comp->airOutletModelObject();
+        } else if (auto comp = modelObject->optionalCast<ZoneHVACComponent>()) {
+          modelObjects.insert(modelObjects.begin(), *comp);
+          modelObject = comp->outletNode();
```

Comment:
AirLoopHVACUnitarySystem also suffered from the same problem... It was not returned as part of `AirLoopHVACOutdoorAirSystem_Impl::reliefComponents` / `AirLoopHVACOutdoorAirSystem_Impl::oaComponents`